srcpkgs/freerdp3/template

```diff
@@ -0,0 +1,143 @@
+ -DDOCBOOKXSL_DIR=/usr/share/xsl/docbook
+ -DPROXY_PLUGINDIR=/usr/lib/${pkgname}/server/proxy/plugins
+ -DWAYLAND_SCANNER=/usr/bin/wayland-scanner
+ -DXSLTPROC_EXECUTABLE=/usr/bin/xsltproc
+
+ -DPKG_CONFIG_RELOCATABLE=NO
+ -DWITH_ALSA=ON
+ -DWITH_CAIRO=ON
+ -DWITH_CHANNELS=ON
+ -DWITH_CLIENT_CHANNELS=ON
+ -DWITH_CLIENT_SDL=ON
+ -DWITH_CUPS=ON
+ -DWITH_DSP_FFMPEG=ON
+ -DWITH_FFMPEG=ON
+ -DWITH_FREERDP_DEPRECATED_COMMANDLINE=ON
+ -DWITH_FUSE=ON
+ -DWITH_GSTREAMER_1_0=ON
+ -DWITH_ICU=ON
+ -DWITH_JPEG=ON
+ -DWITH_PCSC=ON
+ -DWITH_PULSE=ON
+ -DWITH_SERVER=ON
+ -DWITH_SERVER_CHANNELS=ON
+ -DWITH_SWSCALE=ON
+ -DWITH_WAYLAND=ON
+ -DWITH_WINPR_TOOLS=ON
+ -DWITH_X11=ON
+ -DWITH_XCURSOR=ON
+ -DWITH_XEXT=ON
+ -DWITH_XI=ON
+ -DWITH_XINERAMA=ON
+ -DWITH_XKBFILE=ON
+ -DWITH_XRENDER=ON
+ -DWITH_XV=ON
+
+ -DWITH_LIBSYSTEMD=OFF
+ -DWITH_SAMPLE=OFF"
+hostmakedepends="pkg-config wayland-devel xmlto"
+makedepends="alsa-lib-devel cJSON-devel cairo-devel cups-devel ffmpeg-devel
+ fuse3-devel icu-devel libXcursor-devel libXdamage-devel libXinerama-devel
+ libXrandr-devel libXv-devel libjpeg-turbo-devel libusb-devel libxkbcommon-devel
+ libxkbfile-devel mit-krb5-devel openssl-devel pcsclite-devel
+ pkcs11-helper-devel pulseaudio-devel uriparser-devel wayland-devel"
+short_desc="Free RDP (Remote Desktop Protocol) client"
+maintainer="chrysos349 <chrysostom349@gmail.com>"
+license="Apache-2.0"
+homepage="https://www.freerdp.com/"
+changelog="https://raw.githubusercontent.com/FreeRDP/FreeRDP/3.5.0/ChangeLog"
+distfiles="https://github.com/FreeRDP/FreeRDP/archive/${version}.tar.gz"
+checksum=03323b383980ee91decbed88270bac061ffb17fd04e52576c70da7885601ecbe
+CFLAGS="-Wno-dev"
+
+# help cmake generate manuals when cross-compiling
+post_patch() {
+	vsed -i cmake/InstallFreeRDPMan.cmake \
+		-e '/generate_argument_docbook/s/COMMAND/& \${CMAKE_CROSSCOMPILING_EMULATOR}/'
+}
+
+pre_configure() {
+	if [ "$CROSS_BUILD" ]; then
+		export CXX="$XBPS_WRAPPERDIR/${XBPS_CROSS_TRIPLET}-c++"
+		configure_args+="
+		 -DCMAKE_CXX_COMPILER_AR=/usr/bin/${XBPS_CROSS_TRIPLET}-ar
+		 -DCMAKE_CXX_COMPILER_RANLIB=/usr/bin/${XBPS_CROSS_TRIPLET}-ranlib"
+	fi
+}
+
+case "$XBPS_TARGET_MACHINE" in
+	i686*|x86_64*) configure_args+=" -DWITH_SSE2=ON";;
+	armv5tel*) configure_args+=" -DWITH_NEON=OFF";;
+esac
+
+# first we want to separate -server, everything else then goes to client
+subpackages="libfreerdp-server3 freerdp-server3 freerdp-server3-devel
+ libfreerdp3 freerdp3-devel"
+
+libfreerdp3_package() {
+	short_desc+=" - runtime libraries"
+	pkg_install() {
+		vmove "usr/lib/*.so.*"
+	}
+}
+
+freerdp3-devel_package() {
+	short_desc+=" - development files"
+	depends="cJSON-devel libfreerdp3>=${version}_${revision}
+	 openssl-devel"
+	pkg_install() {
+		vmove "usr/lib/*.so"
+		vmove usr/include
+		vmove usr/lib/cmake
+		vmove usr/lib/pkgconfig
+	}
+}
+
+libfreerdp-server3_package() {
+	short_desc="Free RDP (Remote Desktop Protocol) server - runtime files"
+	depends="libfreerdp3>=${version}_${revision}"
```

Comment:
I assume that the libraries in this package will pull in `libfreerdp3` automatically with shlib dependencies. If so, you can drop this line.